polyply/src/backmap.py

```diff
@@ -98,16 +98,14 @@ def orient_template(meta_molecule, current_node, template, built_nodes):
     opt_coords = np.zeros((3, len(edges)))
 
     for ndx, edge in enumerate(edges):
-        resids = []
         for atom in edge:
             resid = meta_molecule.molecule.nodes[atom]["resid"]
             is_current = resid == current_resid
-            resids.append(resid)
```

Comment:
resids undefined here?
And, `is_current` is still an unused variable.

Reply:
OK must have missed this or wanted to fix this.